src/components/Card/CardToolbar.jsx

```diff
@@ -2,35 +2,29 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import omit from 'lodash/omit';
 import { Close16, Popup16 } from '@carbon/icons-react';
-import { OverflowMenu, OverflowMenuItem } from 'carbon-components-react';
+import { OverflowMenu, OverflowMenuItem, Button } from 'carbon-components-react';
 import classNames from 'classnames';
 
 import { settings } from '../../constants/Settings';
 import { CARD_ACTIONS } from '../../constants/LayoutConstants';
 
 import CardRangePicker, { CardRangePickerPropTypes } from './CardRangePicker';
 
-const { iotPrefix } = settings;
+const { iotPrefix, prefix } = settings;
 
 const ToolbarSVGWrapper = props => {
-  const { children, onClick } = props;
   return (
-    <button
-      type="button"
-      onClick={onClick}
+    <Button
+      kind="ghost"
       className={classNames(
         `${iotPrefix}--card--toolbar-action`,
-        `${iotPrefix}--card--toolbar-svg-wrapper`
+        `${iotPrefix}--card--toolbar-svg-wrapper`,
+        `${prefix}--btn--icon-only` // can't actually use the hasIconOnly prop because we don't want the tooltip
```

Comment:
Is there some context on why the tooltip isn't desired? It's an a11y affordance that I think we should give for these icon buttons - [some context from Carbon](https://github.com/carbon-design-system/carbon/issues/5069#issuecomment-576375526)

Reply:
it's how greedy the iconOnly tooltip is @tay1orjones , we've requested a delay be added so we can keep from popping the tooltip too often for those that already understand what this Icon is for: https://github.com/carbon-design-system/carbon/issues/6178.  We are adding the browser tooltip though for accessibility so if you hover you'll see the in-browser version

Reply:
the focus stealing is actually a carbon issue, I've opened this to track: https://github.com/carbon-design-system/carbon/issues/6227